Source/Core/Core/FreeLookManager.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/FreeLookManager.h"
+
+#include "Common/Common.h"
+#include "Common/CommonTypes.h"
+#include "Common/Config/Config.h"
+
+#include "Core/ConfigManager.h"
+#include "Core/FreeLookConfig.h"
+
+#include "InputCommon/ControllerEmu/ControlGroup/Buttons.h"
+#include "InputCommon/InputConfig.h"
+
+#include "VideoCommon/FreeLookCamera.h"
+#include "VideoCommon/OnScreenDisplay.h"
+
+namespace
+{
+enum MoveButtons
+{
+  Up,
```

Comment:
These aren't enum classes, so each member should be prefixed with the name of the enum (`MoveButtons_Up` instead of `Up` for example). Otherwise this introduces `Up`, `Down`, etc. into file scope and it's not clear what they are :)

Alternatively, wrap them inside a namespace (e.g. namespace MoveButtons) so that referring to them requires typing MoveButtons:: like with enum classes.

Reply:
Makes sense, just ugly.  I guess I'll switch to the namespace approach, thanks!